arcgis-ios-sdk-samples/Scenes/Surface placements/SurfacePlacementsViewController.swift

```diff
@@ -17,65 +17,128 @@
 import UIKit
 import ArcGIS
 
-/// A view controller that manages the interface of the Surface Placements
-/// sample.
+/// A view controller that manages the interface of the Surface placements sample.
```

Comment:
The word "sample" was on a separate line so that the comment wrapped at the 80 column page guide:

![image](https://user-images.githubusercontent.com/2257493/86976714-e8b4ad00-c12f-11ea-90da-e102303d4c69.png)

Also, I used title case in these documentation comments because there's a word (sample) that appears after the title, making it really confusing as to where the title ends.

Reply:
Ahh I see. How can I config Xcode to let it display the 3-slash comment as monospace font? 🤔 

![image](https://user-images.githubusercontent.com/9660181/86981499-9c6f6a00-c13b-11ea-9eb2-d3097e4b8a88.png)

---

Personally I'm all for a better coding style and format. While 80 column line wrap is ideal for a nice looking codebase, it occasionally drives me nut to find ways to cut short a line 🌀  (e.g. add newlines to function params, flatten condition/callback pyramid by extracting closures into functions, etc.). 

And since `SwiftLint` linter has a 40-line limit for a single function, when refactoring previous samples, I run into cases that trigger linter rule if I add a few long lines of code. And I need to find ways to bypass it, such as 

https://github.com/Esri/arcgis-runtime-samples-ios/blob/02231f0af8e174676968d17d96547a2e6bc67905/arcgis-ios-sdk-samples/Utility%20Network/Display%20utility%20associations/DisplayUtilityAssociationsViewController.swift#L151

So my thought is to loosen line break to 100-column width, considering the sometimes quite long API names in Swift.
@vquach2404 what's your thought? Or how do we wrap longer lines in ObjC codebase? 😳

Reply:
To be clear, I only wrap documentation comments at the page guide so that they stay at a nice readable width. I haven't tried nor do I see any reason to start trying to wrap code at a page guide. That would just be nuts! 🥜

Reply:
I see!